n2y/main.py

```diff
@@ -157,11 +157,23 @@ def main(raw_args, access_token):
     return 0
 
 
+def get_database_pages_and_prefetch_blocks(database):
```

Comment:
Let me know what you think about this @johndgiese. I could just load the block in the Page constructor but I figured there must be a reason why we're lazy-loading it (hence the pre-fetching).